src/test/scala/chisel3/internal/NameCollisionSpec.scala

```diff
@@ -0,0 +1,80 @@
+// SPDX-License-Identifier: Apache-2.0
+
+package chiselTests
+
+import chisel3._
+import chisel3.stage.ChiselStage
```

Comment:
```suggestion
import circt.stage.ChiselStage
```
We're trying to avoid `chisel3.stage.ChiselStage`